packages/nettacker/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nettacker
+_pkgname=Nettacker
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanners')
+url='https://github.com/OWASP/Nettacker'
+license=('Apache')
+depends=('flake8' 'python-backoff' 'python-argparse' 'python-netaddr' 'python-ipaddr' 'python-requests' 'python-paramiko' 'python-texttable' 'python-pysocks' 'python-win_inet_pton' 'python-pyopenssl' 'python-flask' 'python-lockfile' 'python-bs4' 'python-pyspf' 'python-sqlalchemy' 'canari' 'python-py3dns' 'python-censys' 'python-six' 'python-shodan' 'python-pycurl' 'python-xmltodict' 'python-rule' 'python-mechanize' 'scapy' 'python-psycopg2')
+makedepends=('python-setuptools' 'git')
+source=("git+https://github.com/OWASP/$_pkgname.git")
```

Comment:
github URL are case unsensitive so https://github.com/OWASP/$_pkgname.git == https://github.com/OWASP/$pkgname.git, so in the end there is no need for `_pkgname`

Reply:
i use it to change my current directory, the directory called Nettacker not nettacker

Reply:
if you git clone https://github.com/OWASP/nettacker.git the directory will be named nettacker so cd $pkgname will work

Reply:
oh , got it. thanks